### PR TITLE
W-13690648 AEH: doc improvement for connecting to Salesforce jh

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

File: modules/ROOT/pages/connecting-to-salesforce.adoc [hunk 1]
```diff
@@ -3,13 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API Experience Hub has a Salesforce organization with Experience Cloud and a managed package that is installed in this organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization.
+API Experience Hub has a Salesforce organization with Experience Cloud and a managed package. The managed package is installed during the onboarding process when the API Experience Hub administer connects their Anypoint Platform organization to their Salesforce Organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization. The Salesforce System Administrator can view the *API Experience Hub* package in their Salesforce organization from Salesforce by navigating to *Setup* > *Installed Packages* and selecting the *API Experience Hub Package*.
```
Comment (sathya0) [inline modules/ROOT/pages/connecting-to-salesforce.adoc]:
```suggestion
API Experience Hub has a Salesforce organization with Experience Cloud and a managed package. The managed package is installed during the onboarding process when the API Experience Hub administrator connects their Anypoint Platform organization to their Salesforce organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization. The Salesforce System Administrator can view the *API Experience Hub* package in their Salesforce organization from Salesforce by navigating to *Setup* > *Installed Packages* and selecting the *API Experience Hub Package*.
```

Comment (sathya0) [inline modules/ROOT/pages/connecting-to-salesforce.adoc]: selecting **API Experience Hub Package** or selecting the **API Experience Hub** package ?

File: modules/ROOT/pages/connecting-to-salesforce.adoc [hunk 2]
```diff
@@ -3,13 +3,9 @@ ifndef::env-site,env-github[]
...
 
-A connection with Salesforce Experience Cloud is required for linking to the Salesforce organization to create an API portal within Anypoint. This process must be completed only once to create a portal. 
-
-IMPORTANT: Ensure that you link your Anypoint organization to the correct Salesforce organization. 
-
-An instance of API Experience Hub connects to only one Anypoint Platform organization and one Salesforce organization.
+A connection with Salesforce Experience Cloud is required for linking to the Salesforce organization to create an API portal within Anypoint. This process must be completed only once to create a portal. Currently, an instance of API Experience Hub can connect only to one Anypoint Platform organization and one Salesforce organization.
```
Comment (sathya0) [inline modules/ROOT/pages/connecting-to-salesforce.adoc]: style guide says not to use just Anypoint

**Review by sathya0 — DISMISSED** [review]

File: modules/ROOT/pages/connecting-to-salesforce.adoc (hunk 2 above)
Comment (JennyHajee) [reply modules/ROOT/pages/connecting-to-salesforce.adoc]: Thanks, fixed

File: modules/ROOT/pages/connecting-to-salesforce.adoc (hunk 1 above)
Comment (JennyHajee) [reply modules/ROOT/pages/connecting-to-salesforce.adoc]: API Experience Hub Package

**Review by sathya0 — APPROVED** [review]

**Pull request merged**